core/templates/dev/head/collection_editor/CollectionEditorNavbarBreadcrumbDirective.js

```diff
@@ -0,0 +1,66 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Primary controller for the collection editor navbar breadcrumb.
+ *
+ * @author oskar.cieslik@gmail.com (Oskar Cieslik)
+ */
+
+oppia.directive('collectionEditorNavbarBreadcrumbDirective', [function() {
+  return {
+    restrict: 'E',
+    templateUrl: 'inline/collection_editor_navbar_breadcrumb_directive',
+    controller: ['$scope', 'WritableCollectionBackendApiService',
+      'CollectionRightsBackendApiService', 'CollectionObjectFactory',
+      'CollectionUpdateService', 'UndoRedoService', 'warningsData',
+      'routerService', 'CollectionBackendApiService', function(
+      $scope, WritableCollectionBackendApiService,
+      CollectionRightsBackendApiService, CollectionObjectFactory,
+      CollectionUpdateService, UndoRedoService, warningsData, routerService,
+      CollectionBackendApiService) {
+      $scope.collection = null;
```

Comment:
Well, I figured it out by wrapping CollectionEditorNavbarBreadcrumbDirective in ng-controller="CollectionEditor". I had problem with it before, because I didn't notice I overwritten getCollection() method below 😅. Is this the correct approach to have the same controller in two places? I think that shared service would be better for this, what do you think? :)